CHANGES.txt

```diff
@@ -45,6 +45,11 @@ Breaking Changes
 Changes
 =======
 
+- Fix the order by push down optimisation for ``join`` statements that order by
```

Comment:
I would phrase it simpler for the end user: `Fixed and issue that could cause incorrect results for a ``join`` query with order by due to push down optimisation.`